api/src/controllers/utils.ts

```diff
@@ -163,6 +163,24 @@ router.post(
 	respond
 );
 
+router.post(
+	'/transform-export',
+	asyncHandler(async (req, res, next) => {
+		if (req.is('application/json') === false || !req.headers['content-type']) {
+			throw new InvalidPayloadException(`Payload should be a JSON object`);
+		}
+
+		if (!req.sanitizedQuery.export) {
+			throw new ForbiddenException();
```

Comment:
I think Forbidden may not be the most appropriate error here, `InvalidQuery` perhaps

Reply:
Refactored, `InvalidQuery` should be it. Thanks! 🙂